client/src/components/core/ms-modal/MsTextInputModal.vue

```diff
@@ -3,6 +3,7 @@
 <template>
   <ms-modal
     :title="$props.title"
+    :subtitle="$props.subtitle"
```

Comment:
why do you need to use $props, it should work with only subtitle.

Reply:
Yeah it does but I like to be explicit. I don't know what happens if I create a variable in my component named `subtitle`. This way it's explicit, I know where the variable is from, and I avoid potential conflicts.

Reply:
.

Reply:
Removed the `$props`, see #5488